src/renderer/services/ethereum/transaction.ts

```diff
@@ -45,7 +45,7 @@ export const createTransactionService = (client$: Client$): TransactionService =
     )
 
   const runIsApprovedERC20Token$ = (client: EthClient, params: ApproveParams): LiveData<ApiError, boolean> =>
-    Rx.from(client.isApproved(params.spender, params.sender, params.amount || baseAmount(0))).pipe(
+    Rx.from(client.isApproved(params.spender, params.sender, params.amount || baseAmount('1'))).pipe(
```

Comment:
Why  `0`  -> `1` ?

Reply:
because xchain-eth compares using lte
0 <= 0 so it always return true even though it wasn't approved.
So keep it above one or need to fix xchain-eth `isApproved` function to use lt rather than lte for bignumber compare

Reply:
Ok, keep `baseAmount(1)` then.